### ISSUE TITLE
ubuntu16: recover fails due shrinking disk size

### ISSUE THREAD

**Issue by gdha** [gdha/rear-automated-testing#23]
````
++ lvm pvcreate -ff --yes -v --uuid n9nk2f-WzLi-OMxx-rg3I-N1S8-A6sY-gEMZmv --restorefile /var/lib/rear/layout/lvm/ubuntubox-vg.cfg
/dev/sda5
  /run/lvm/lvmetad.socket: connect failed: No such file or directory
  WARNING: Failed to connect to lvmetad. Falling back to internal scanning.
  Couldn't find device with uuid n9nk2f-WzLi-OMxx-rg3I-N1S8-A6sY-gEMZmv.
    Wiping cache of LVM-capable devices
  Physical extents end beyond end of device /dev/sda5.
  Format-specific initialisation of physical volume /dev/sda5 failed.
  Failed to setup physical volume "/dev/sda5"
````
I noticed that the Disk /dev/sda: 322GB is huge and only used little, perhaps the box **yk0/ubuntu-xenial** is not the proper choice?

**Comment by gdha** [comment]
The idea was to have a box that could work with virtualbox and libvirt - yk0/ubuntu-xenial - but I might just test **ubuntu/xenial64** (which is for virtualbox only).
Gonna try **boxcutter/ubuntu1610** also.
Both boxes are having issues with:
````
`playbook` does not exist on the guest: /vagrant/ansible/common/vagrant-playbook.yml
````

**Comment by gdha** [comment]
The fix was using the **bento/ubuntu-16.04** one and some change in the **Vagrantfile**.
